### PR TITLE
Touch Framework Shell Script

### LINKED ISSUE #1: Dynamic framework rebuild needs to be cleaned
When the framework is static, you can change Kotlin, rebuild in Xcode, and the change will be seen. When the framework is dynamic, Kotlin will be rebuilt, but you'll only see the change when you clean the build folder.

Ideally changing Kotlin and running won't need a clean build.

### PR DESCRIPTION
**Pull request by ben-whit-ley**
Now generates a podspec with an additional script phase that will reload the framework to check for and pull in new Kotlin code.

I think there's room for improvement on the script. Ideally it would only touch the shared Kotlin library. Currently it touches all frameworks.

### CODE REVIEW

**Comment by russhwolf** [comment]
Nit: the new code is using tabs where the stuff around it is using spaces

We talked in huddle but for the sake of having it in writing, it doesn't seem like a dealbreaker to touch all frameworks instead of just the kotlin one, since the alternative is to do a clean that rebuilds everything anyway.

File: gradle-plugin/src/main/kotlin/co/touchlab/kotlin/gradle/targets/native/tasks/CocoapodsTasks.kt [hunk 1]
```diff
@@ -103,7 +103,13 @@ open class PodspecTask : DefaultTask() {
             |                    -P${KotlinCocoapodsPlugin.HEADER_PATHS_PROPERTY}="${'$'}HEADER_SEARCH_PATHS" \
             |                    -P${KotlinCocoapodsPlugin.FRAMEWORK_PATHS_PROPERTY}="${'$'}FRAMEWORK_SEARCH_PATHS"
             |            SCRIPT
-            |        }
+            |        },
+			|        {
+			|        	:name => 'Touch $specName.framework',
+			|        	:execution_position => :after_compile,
+			|        	:shell_path =>  '/bin/sh',
+			|        	:script => 'find "${'$'}{SRCROOT}" -type f -name *frameworks.sh -exec bash -c "touch \"{}\"" \;'
```
Comment (kpgalligan) [inline gradle-plugin/src/main/kotlin/co/touchlab/kotlin/gradle/targets/native/tasks/CocoapodsTasks.kt:111]: I'm pretty sure we'll get some issue reports on this, but let's see how it goes.

**Review by kpgalligan — APPROVED** [review]

**Comment by kpgalligan** [comment]
> Nit: the new code is using tabs where the stuff around it is using spaces
> 
> We talked in huddle but for the sake of having it in writing, it doesn't seem like a dealbreaker to touch all frameworks instead of just the kotlin one, since the alternative is to do a clean that rebuilds everything anyway.

We used to have a team code style to avoid code review issues, and some projects have (or should have) a project style. We should get back to that. However, for this PR, I'm not super concerned.

**Comment by kpgalligan** [comment]
> We talked in huddle but for the sake of having it in writing, it doesn't seem like a dealbreaker to touch all frameworks instead of just the kotlin one, since the alternative is to do a clean that rebuilds everything anyway.

Agree. Maybe we can improve later. Merge?

**Comment by kpgalligan** [comment]
I'm merging.

**Pull request merged**